tools/functional-tester/etcd-tester/main.go

```diff
@@ -111,17 +117,21 @@ func main() {
 			schedule[i] = failures[caseNum]
 		}
 	}
-
+	checkers := make([]Checker, 0)
 	t := &tester{
 		failures: schedule,
 		cluster:  c,
 		limit:    *limit,
-		checker:  newNoChecker(),
+		checkers: checkers,
```

Comment:
I think there should be a composite checker that implements the Checker instead of passing a list of checkers around. Keeping a list means that the tester now has to reason about multiple checkers despite it being much simpler to reason about a single checker.